Sources/Components/TimeLineView/DottedTimeLineIndicatorProvider.swift

```diff
@@ -0,0 +1,78 @@
+import FinniversKit
+
+public struct DottedTimeLineIndicatorProvider: TimeLineIndicatorProvider {
+    private static let textVerticalMargin: CGFloat = .spacingXS
+
+    public let width: CGFloat = DottedTimeLineIndicatorView.indicatorDotSize
+
+    private let font: UIFont
+    private var dotOffset: CGFloat = 0
+    private var fontHeight: CGFloat {
+        "I".height(withConstrainedWidth: .greatestFiniteMagnitude, font: font)
```

Comment:
Unless there is something special with measuring the "I", this is the same as the capHeight property of the font.

Reply:
I had no idea about that property😄 Thanks!